### DISCUSSION TITLE
control sidebar options ?

### DISCUSSION THREAD

**Discussion by vikas-kumar-tuespot** [ColorlibHQ/AdminLTE#3261] [Q&A] [answered]
how to fix changes in control sidebar so that it's work same after reload page?

**Comment by REJack** [comment, marked as answer]
For this you need to rework the demo.js, the demo.js is only for a demo purposes. You shouldn’t use the demo.js in production.

**Reply by vikas-kumar-tuespot** [reply]
thanks for replay

**Reply by vikas-kumar-tuespot** [reply]
and ill do it.
